test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -195,6 +195,9 @@ object Assertion {
   final def contains[A](element: A): Assertion[Iterable[A]] =
     Assertion.assertion("contains")(param(element))(_.exists(_ == element))
 
+  final def include(element: String): Assertion[String] =
```

Comment:
`hasSubstring`? Whatever name we choose, we should put it in alphabetical order with the other assertions and I would make the string description of the assertion match its name (e.g. if we go with `hasSubstring` it would be `Assertion.assertion("hasSubstring")`. I would also add a couple of tests for this in `AssertionSpec`, one for failure and one for success, along the lines of the other tests there.